modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -718,6 +718,57 @@ definition {
 			actionName = "Update Action Name");
 	}
 
+	@description = "LPS-156346 - Assert an action with an Expression using Groovy Script will resolve."
```

Comment:
Hi @epark18 I think this description would be: `LPS-152181 - Verify that the expression works with Groovy Script`

cc: @timpak

Reply:
Thanks for the suggestions @evanilson-santana ! Updated